acquire/collector.py

```diff
@@ -179,10 +179,11 @@ class Collector:
     METADATA_BASE = "$metadata$"
     COMMAND_OUTPUT_BASE = f"{METADATA_BASE}/command-output"
 
-    def __init__(self, target: Target, output: Output, base="fs"):
+    def __init__(self, target: Target, output: Output, base: str = "fs", skip_list: set = set()):
```

Comment:
`set` is a mutable object, should the initialization not be moved to the assignment to `self`?

```python
def __init__(self, target: Target, output: Output, base: str = "fs", skip_list: set = None):
    ...
    self.skip_list = skip_list or set()
```

Reply:
Good one